CMakeLists.txt

```diff
@@ -2498,20 +2498,20 @@ function(install_manpage_symlink SOURCE TARGET MANDIR)
         # If we haven't found an ln executable with MinGW, we don't try
         # generating and installing the man pages, so if we get here,
         # we've found that executable.
-        set(LINK_COMMAND "\"${LINK_EXECUTABLE}\" \"-s\" \"${SOURCE}\" \"${TARGET}\"")
+        set(LINK_COMMAND "\"${LINK_EXECUTABLE}\" \"-s\" \$ENV{DESTDIR}\"${SOURCE}\" \"${TARGET}\"")
     else(MINGW)
-        set(LINK_COMMAND "\"${CMAKE_COMMAND}\" \"-E\" \"create_symlink\" \"${SOURCE}\" \"${TARGET}\"")
+        set(LINK_COMMAND "\"${CMAKE_COMMAND}\" \"-E\" \"create_symlink\" \$ENV{DESTDIR}\"${SOURCE}\" \"${TARGET}\"")
     endif(MINGW)
 
     install(CODE
         "message(STATUS \"Symlinking: ${CMAKE_INSTALL_PREFIX}/${MANDIR}/${SOURCE} to ${TARGET}\")
```

Comment:
You should prepend it to `CMAKE_INSTALL_PREFIX` here as well, so that the progress messages for adding symlinks are correct.

Reply:
Done